util/regexp-assemble/data/932130.data

```diff
@@ -1,8 +1,10 @@
-##! Please refer to the documentation at 
+##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regexp_assemble/.
 
 \$\(.*\)
 \$\{.*\}
 <\(.*\)
 >\(.*\)
 \$\(\(.*\)\)
+/\w*\[!.+\]
+/\w*\[.+\]
```

Comment:
Do we want to add a comment with the intended matches on this part? I think it would be very useful for future 👀

Reply:
Added comments.

Reply:
Awesome!